gpg_import.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+### SOURCE 
+. <(xargs -1 bash -c 'printf "export %q\n" "$@"' -- < /proc/1/environ)
+export GPG_TTY=$(tty)
+
+gpg --list-keys
+#mkdir /root/.gnupg
+#chmod 700 /root/.gnupg
+mv gpg_public.asc /root/.gnupg/
+mv gpg_private.asc /root/.gnupg/
+### import the gpg public key
+echo "import the gpg public key"
+gpg --import /root/.gnupg/gpg_public.asc
+echo "import the private key"
+gpg --batch --passphrase="test1234" --import /root/.gnupg/gpg_private.asc
```

Comment:
should not need to import the private key

Reply:
duplicity gives an error without the private key and stops backing up

Reply:
ahhh, no it looks like because i had some old metadata on the old bucket. Every time you re-deploy the container, it looks like its being considered as old metadata and is now looking for private key to decrypt the old one before it could do anything.

Reply:
it should be able to use a local metadata cache

Reply:
gpg_public.asc is no longer needed. This is coming from the HOST/docker env now rather than copied to a file from the repo.